CreeDictionary/API/apps.py

```diff
@@ -37,14 +34,13 @@ def ready(self) -> None:
     def perform_time_consuming_initializations(self):
         logger.debug("loading affix caches")
         from API.search import affix
+        from API.models import WordformCache
 
         affix.cache.preload()
 
-        logger.debug("calling initialize_preverb_search()")
-        initialize_preverb_search()
-
-        logger.debug("calling read_morpheme_rankings()")
-        read_morpheme_rankings()
+        # Accessing these cached properties will preload them
+        WordformCache.PREVERB_ASCII_LOOKUP
+        WordformCache.MORPHEME_RANKINGS
```

Comment:
Not necessary, but this matches the `affix.cache.preload()` line from above:

```suggestion
        WordformCache.preload()
```